### PR TITLE
candi local.cfg: add netcdf

### PR DESCRIPTION
**Pull request by tjhei**
part of #5447

### CODE REVIEW

**Review by MFraters — APPROVED** [review]
Looks good, but shouldn't netcdf also be added to the `doc/sphinx/user/install/local-installation/using-candi.md` optional packages which may be required in the future? Or do we just want to update the text to require both?

**Comment by tjhei** [comment]
> optional packages which may be required in the future? Or do we just want to update the text to require both?

We are talking about making sundials and netcdf mandatory in the near future. But you are right. I think I addressed your comment.

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
The indent check fails.

File: doc/sphinx/user/install/local-installation/using-candi.md [hunk 1]
```diff
@@ -34,6 +32,15 @@ class="smallcaps">p4est</span> and Trilinos.
     configuration files in that directory, named `candi.cfg` and `local.cfg`).
     When both `candi.cfg` and `local.cfg` are present in the directory, running
     candi (see below) will by default use the configuration options in `local.cfg`.
+    
+    If you want to work without the `local.cfg`, be aware that you should enable
+    - p4est
+    - trilinos
+    - hdf5 (optional)
+    - netcdf (optional)
+    - sundials (optional)
```
Comment (bangerth) [inline doc/sphinx/user/install/local-installation/using-candi.md:41]: I couldn't quite figure out what the "(optional)" part here means. You're suggesting in the text above that one should enable these. If that's your suggestion, then just remove the qualification, or at least explains what happens if one takes the optional at face value and doesn't do it.

Comment (tjhei) [reply doc/sphinx/user/install/local-installation/using-candi.md]: Fair enough. Updated.

**Review by bangerth — APPROVED** [review]

**Pull request merged**